InAppSettingsKit/Controllers/IASKMultipleValueSelection.m

```diff
@@ -11,6 +11,15 @@ @implementation IASKMultipleValueSelection {
 
 @synthesize settingsStore = _settingsStore;
 
+- (instancetype)initWithSettingsStore:(id<IASKSettingsStore>)settingsStore
+{
+    self = [super init];
+    if (self) {
+        self.settingsStore = settingsStore;
```

Comment:
Happy to merge your fix. For consistency with the rest of the code, can you change the init syntax like this? 

    (id)initWithSettingsStore:(id<IASKSettingsStore>)settingsStore {
        if ((self = [super init])) {
        …

That would be great. Thanks!